datus/resources/skills/create-skill/SKILL.md

```
@@ -96,15 +96,15 @@ skill-name/
 
 ## Scaffold the Directory
 
-Use `skill_*` tools (paths relative to skills directory root):
+Use `write_file` from the filesystem tools. Paths must start with `.datus/skills/` (project-level) or `~/.datus/skills/` (user-level) — see Critical Rule #2:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)